src/main/java/io/lettuce/core/dynamic/SimpleBatcher.java

```diff
@@ -148,12 +148,12 @@ private List<RedisCommand<Object, Object, Object>> prepareForceFlush() {
 
         List<RedisCommand<Object, Object, Object>> batch = new ArrayList<>(Math.max(batchSize, 10));
 
-        do {
+        while (!queue.isEmpty()) {
+
             RedisCommand<Object, Object, Object> poll = queue.poll();
 
-            assert poll != null;
```

Comment:
I am removing this `assert` because it doesn't help with production code. I searched for uses of `assert` throughout the code and found only this and another one a few lines below, which I'm also removing.